Alcolator/Alcolator/AppDelegate.m

```diff
@@ -7,6 +7,7 @@
 //
 
 #import "AppDelegate.h"
+#import "Beer.h"
```

Comment:
Since we don't use this in the AppDelegate, I would leave this `import` statement out, though I know you'd been considering initializing your `Beer` class in the `didFinishLaunchingWithOptions:` method, but I don't think that's the right place, either.